lib/math.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var R = require('ramda');
+
+var operators = {
+  '+': R.add,
+  '-': R.subtract,
+  '*': R.multiply,
+  '/': R.divide,
+  '%': R.modulo,
+  '++': R.inc,
+  '--': R.dec
+};
```

Comment:
Also, would it be silly to support `**` for the sake of purism?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Exponentiation_(**)